### ISSUE TITLE
wrong filter in /administrator/components/com_mycomps/models/forms/mycomp.xml

### ISSUE THREAD

**Issue by ghazal** [Niambie/jfoobar#21]
hi,
last version downloadable
 ie Niambie-jfoobar-e03b13a

Since the last 2 updates, I get the wrong filter in
/administrator/components/com_mycomps/models/forms/mycomp.xml
± lines 45 and 54

```
   <field
       name="snippet"
       type="editor"
       class="inputbox"
       label="COM_MYCOMPS_FIELD_SNIPPET_LABEL"
       description="COM_MYCOMPS_FIELD_SNIPPET_DESC"

      ------- >  filter="ContactsHelper::filterText"
```

-----> instead of ---> filter="MycompsHelper::filterText"

```
       buttons="false" />

   <field
       name="fulltext"
       type="editor"
       class="inputbox"
       label="COM_MYCOMPS_FIELD_FULLTEXT_LABEL"
       description="COM_MYCOMPS_FIELD_FULLTEXT_DESC"
        ------- > filter="ContactsHelper::filterText"
       buttons="true" />
```

**Comment by ghazal** [comment]
Hi again,
after updating to j!1.7.1, error still occurring.

**Comment by BloodBaz** [comment]
I have fixed the code and submitted a Pull Request (Issue #28).  Hopefully Amy will update soon.

**Comment by mbabker** [comment]
Fixed in #28
